pywal/__main__.py

```diff
@@ -69,7 +69,7 @@ def get_args(args):
                      help="Print \"wal\" version.")
 
     arg.add_argument("-e", action="store_true",
-                     help="Skip Reloading Environment gtk/xrdb/i3/polybar")
+                     help="Skip Reloading Environment gtk/xrdb/i3/sway/polybar")
```

Comment:
This line is one character too long. Rename `Environment` to `Programs`.

Reply:
Does it even need to have a word? I think "Skip reloading gtk/xrdb/i3/sway/polybar" is clear enough.